scripts/scil_visualize_multi_shell.py

```diff
@@ -70,6 +70,15 @@ def main():
     args = parser.parse_args()
     assert_inputs_exist(parser, args.scheme_file)
 
+    if len(args.scheme_file) == 2:
+        assert_gradients_filenames_valid(parser, args.scheme_file, 'fsl')
+    elif len(args.scheme_file) == 1:
+        assert_gradients_filenames_valid(parser, args.scheme_file, 'mrtrix')
+    else:
+        parser.error('Depending on the gradient format you should have '
+                     'two files for FSL format and one file for MRtrix')
+
+    out_basename = None
     if args.out:
         out_basename, ext = split_name_with_nii(args.out)
```

Comment:
Since the output is expected to be a `.png`, I think being robust to `.gz` is useless